### DISCUSSION TITLE
transform changes don't cause movement after the parent entity is despawned

### DISCUSSION THREAD

**Discussion by or-n** [bevyengine/bevy#11033] [Q&A]
I have an enemy in defender clone and add a person bundle to it. When it it hit, all its children which are found in the person_query have the state set to 'falling' and the enemy is despawned. After that transform changes don't affect the person

**Comment by Davier** [comment]
If I understand correctly, you despawn an entity, and then modify its children's `Transform`, but they don't move.
The children probably still have their `Parent` component. Since their parent is invalid, their `GlobalTransform`s are no longer updated.
You have to remove the `Children`/`Parent` components when despawning the entity:
```rust
commands.entity(parent).clear_children().despawn();
```
However, when you do so, the children will set their global transform to their local transform, which is probably not what you want. You will need to update their transform:
```rust
*transform = global_transform.reparented_to(GlobalTransform::default());
```

**Reply by or-n** [reply]
I'd probably do that, but now I see another issue. The parent entity scale is inherited and I don't want that
